README.md

```diff
@@ -19,74 +19,94 @@ These are related projects that are either already setup and available for use o
 
-#### Adding private SSH key to your VM
+#### Adding a private SSH key to your VM
 
-Add your private ssh key under `~/.ssh/<your_key>.pem`, this is typically the same key that you use to SSH to your host VM, regardless it needs to be a key created on the AWS console so Amazon is aware of it. Then do `chmod 400 ~/.ssh/<your_key>.pem` so your key is not publicly viewable.
+Add your private ssh key under `~/.ssh/<your_key>.pem`, this is typically the same key that you use to SSH to your host VM, regardless it needs to be a key created on the AWS console so Amazon is aware of it. Then set privileges to _read-by-user-only_ by `chmod 400 ~/.ssh/<your_key>.pem` so your key is not publicly viewable.
 
-#### TODO:
 
-* Guide on choosing AWS instance type... make sure it matches your AMI.
-* AMI, use an ubuntu 16.04 base box, you can use the official Ubuntu release.  You may need to make your own AMI with more storage! Needs to be in your region!  You may want to google to start with the official Ubuntu images for your region.
+## Installing the Platform (CGP)
 
-### Setup for Boardwalk
+These directions below assume you are using AWS.  We will include additional cloud instructions as `cgp-deployment` matures.
 
-See the Boardwalk [README](boardwalk/README.md) for details.
+
+### Collecting Information
+
+The installation script (`install_bootstrap`) will prompt for several questions. To expedite the installation process it is useful to prepare for some of the answers beforehand. Here are a few pointers:
+
+* make sure you know what AWS region your VM runs in (e.g. `us-west-2`)
+* decide whether you want to create an instance for development or production as it might impact the size and therefore the cost of the host virtual machine (see above for recommendations)
+* find out whether your favorite editor is installed on the VM
+* create a static IP address for your VM (AWS calls this _Elastic IP_); find a short set of instructions above
+* you will be asked to provide an host domain that points to your EC2 instance; you need to know the name of the domain but at the time of installation the domain (or _record set_) does not have to be configured in _Route 53_
+* _Boardwalk_ has functionality to export metadata to Broad's FireCloud; in order to use it you need to provide your Google Cloud Platform credentials; specifically you need the Google Client ID and the Google Client Secret (it's okay to leave the Google site verfication code empty)
+* all metadata reside in an Elasticsearch database; make sure you have the domain name of that Elasticsearch instance handy
+* have the domain name of the _dos-dss server_ handy
 
 ### Running the Installer
 
-Once the above setup is done, clone this repository onto your server and run the bootstrap script.
+Once the above setup is done, clone this repository onto your server and run the bootstrap script. If needed checkout a particular branch or release tag you're interested after you execute the `git clone` command.
```

Comment:
These instructions are particular to the feature/commons branch, and so you should probably mention that branch. While it could be another branch, it would have to be one branched off feature/commons, e.g., trying to deploy develop will not work with these instructions.

Reply:
rephrased it and put it back in